compiler/control/OMROptions.cpp

```diff
@@ -1359,6 +1359,13 @@ OMR::Options::set32BitNumeric(char *option, void *base, TR::OptionTable *entry)
 
 
 char *
+OMR::Options::set32BitNumericInJitConfig(char *option, void *base, TR::OptionTable *entry)
```

Comment:
I have added Doxygen-style comments. 'base' is an unused parameter. It needs to be present because set32BitNumericInJitConfig() is an option processing routine that has a fixed format.